### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by pciavald**
- new gif preview by @allienworks 
- reviewed the text a bit

### CODE REVIEW

**Comment by pciavald** [comment]
@allienworks can you make a download particl image to replace the link ?

**Comment by allienworks** [comment]
> @allienworks can you make a download particl image to replace the link ?

_done_

File: README.md [hunk 1]
```diff
@@ -31,16 +33,17 @@ npm install
 ```
 
 ### Development with Electron
-* Run `ng serve` to start the dev server.
-* Run `npm run start:electron:dev -- -testnet` in another window to start the electron application.
+* Run `npm run start:electron:dev -- -testnet` to start the electron application. Daemon will be updated and launched automatically.
```
Comment (kewde) [inline README.md]: still needs `ng serve` no?

Comment (allienworks) [reply README.md]: I just tried `$ npm run start:electron:fast -- -testnet -debug` and seems like it's not needed anymore (or maybe rather included in the ^)

Comment (kewde) [reply README.md]: That's the fast version, not the dev one tho

Comment (rynomster) [reply README.md]: Yeah true story.. Dev and fast are different.. No live reload with fast

**Comment by kewde** [comment]
Fixed the issues

**Pull request merged**